org.es_de.emulationstation-de.yml

```diff
@@ -0,0 +1,82 @@
+#  SPDX-License-Identifier: MIT
+#
+#  EmulationStation Desktop Edition
+#  org.es_de.emulationstation-de.yml
+#
+#  Flatpak manifest file for use with flatpak-builder.
+#
+
+app-id: org.es_de.emulationstation-de
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+command: emulationstation
+branch: stable
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=host:ro
+  - --filesystem=/var/lib/flatpak:ro
+  - --filesystem=home
+  - --filesystem=~/.var/app
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=PATH=/app/bin:/usr/bin:/var/lib/flatpak/exports/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/var/lib/snapd/snap/bin
```

Comment:
Why this when many of these path don't exist?

Reply:
The application has to be able to locate emulators on various distributions, and this path is a combination of all the paths required based on my thorough testing across these distributions.

Reply:
or you sure that inside the flatpak sandbox there is any of these? Are you really sure?

Reply:
`--filesystem=host` does not give access the `/usr`, `/bin`, `/sbin` among others.

And if you goal is to run host binaries inside the sandbox, then you should revisit this idea.

Reply:
Everything I've attempted to run works with these settings enabled, for example emulators installed as Snaps, Flatpaks and source ports installed to /usr/games/ etc.

Reply:
I doubt. The `/usr` is provided by the faltpak platform and is inside the sandbox. It is unrelated to the distribution.

Reply:
Aha, so you mean that /usr is shadowed by Flatpak itself and can't be accessed by any means from inside the sandbox?

Reply:
Sorry, I mean /usr/bin as for sure /usr/games is accessible as I can launch source ports installed there.

Reply:
There is no `/usr/games` inside the sandbox. and `/usr/bin` is the one from the sandbox.

Reply:
No I don't mean /usr/games is inside the sandbox, but I use the PATH variable to get a list of locations to look for emulators, including /usr/games. I don't necessarily need to have this PATH variable set in the manifest file, but it makes life easier for me since I would otherwise need to hardcode the paths inside the application.

Reply:
Maybe this will make it clearer:
https://gitlab.com/leonstyhre/emulationstation-de/-/blob/master/es-app/src/FileData.cpp

Reply:
this is not what `PATH` is for.

Reply:
The findEmulatorPath() function is where I locate emulators, and it works perfectly fine to find for example those installed in /usr/games or those installed as Flatpaks, Snaps etc. But it requires the PATH information and the security settings in my manifest file to work.

Reply:
The rule set is defined here:
https://gitlab.com/leonstyhre/emulationstation-de/-/blob/master/resources/systems/unix/es_find_rules.xml

Reply:
Setting the PATH using env= provides me with the paths to look for emulators, without this setting there are only two entries defined (I think /usr/bin and /app/bin or something like that?). So I read the PATH variable inside the application to expand the places to look for emulators.

Reply:
I would consider moving it elsewhere, e.g. to a wrapper script, or the application itself.

Reply:
If it's an issue for you I can hardcode it inside the application although it would be a bit ugly. I added these entries to the PATH variable because that's how I look for emulators (well one of multiple ways I look for emulators) so it would require no code changes on my side. If you could confirm that using the environment variables is not acceptable then I'll change, but I'm not sure I understand the concern as it's not a security risk to add entries to the PATH variable?

Reply:
And btw, thanks for fixing the aarch64 build!

Reply:
Thinking more about the PATH variable, the best solution would be if I could access this variable from the "real" system outside the sandbox, then none of this custom or hardcoded stuff would be needed. Is there a reliable way to do this, i.e. reading the PATH of the user that is executing the Flatpak?